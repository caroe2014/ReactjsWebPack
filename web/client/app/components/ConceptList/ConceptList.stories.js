// Copyright Â© 2018-2019 Agilysys NV, LLC.  All Rights Reserved.  Confidential Information of Agilysys NV, LLC.

import React from 'react';
import { storiesOf } from '@storybook/react';
import ConceptList from '.';

storiesOf('ConceptList', module)
  .add('With Images', () => {
    const siteItems = [{
      image: 'https://www.freeiconspng.com/uploads/environment-icon-png-16.png',
      id: '101',
      name: 'Pickle'
    },
    {
      image: 'https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG',
      id: '102',
      name: 'Ice Berg'
    },
    {
      image: 'https://www.freeiconspng.com/uploads/chicken-caesar-salad-png-23.png',
      id: '103',
      name: 'Go Green'
    },
    {
      image: 'https://bakingmischief.com/wp-content/uploads/2016/05/chicken-shawarma-with-yogurt-sauce-image-square1.jpg', // eslint-disable-line max-len
      id: '104',
      name: 'Made Nice'
    }
    ];
    return (
      <ConceptList keyProps={siteItems} />
    );
  })
  .add('Without Images', () => {
    const siteItems = [{
      id: '101',
      name: 'Pickle'
    },
    {
      id: '102',
      name: 'Ice Berg',
      image: 'https://www.freeiconspng.com/uploads/chicken-caesar-salad-png-23.png'
    },
    {
      id: '103',
      name: 'Go Green'
    },
    {
      id: '104',
      name: 'Made Nice'
    }];
    return (
      <ConceptList keyProps={siteItems} />
    );
  });
