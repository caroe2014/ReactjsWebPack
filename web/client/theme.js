// Copyright Â© 2018-2019 Agilysys NV, LLC.  All Rights Reserved.  Confidential Information of Agilysys NV, LLC.

import { css, injectGlobal } from 'styled-components';

const theme = {
  colors: {
    black: '#000000',
    navigationMenu: '#48a942',
    navigationMenuHover: 'cornflowerblue',
    primary: '#347CFF',
    primaryButton: '#347CFF',
    loginField: '#AAA7AC',
    placeHolder: '#909090',
    loginBoxColor: '#434149',
    open: '#48a942',
    close: 'red',
    footer: '#48a942',
    textGrey: '#505050',
    paymentSuccessTitle: '#505050',
    inputBottomBorder: 'lightgray',
    backgroundGrey: '#E6E6E6',
    text: 'black',
    primaryText: '#383838',
    secondaryText: '#a6a6a6',
    hint: '#a6a6a6',
    light: 'white',
    error: '#cc0000',
    tile: '#505050',
    addToCart: '#347CFF',
    itemTile: '#3C474C0',
    darkGrey: '#464646',
    itemBorder: 'lightgrey',
    addOnTab: '#347CFF',
    validationError: '#cc0000',
    loginBackground: '#303030',
    loginFormBackground: '#484848',
    disableBackground: '#E4E4E4',
    addonBg: '#EEEEEE',
    glutenFree: '#8d7356',
    spicy: '#ed1c24',
    vegan: '#0d4a22',
    vegetarian: '#0d4a22',
    new: '#ff6000',
    healthy: '#359919',
    contextBackground: '#f0f0f3',
    loader: '#ababab',
    disableButton: '#D1D1D1'
  },
  fontFamily: 'Roboto',
  receiptFontFamily: 'Roboto',
  fontSize: {
    xl: '2.4rem',
    lg: '1.8rem',
    md: '1.3rem',
    nm: '1rem',
    sm: '0.75rem'
  },
  fontSizeFromProps: (props) => {
    if (props.xl) return props.theme.fontSize.xl;
    if (props.lg) return props.theme.fontSize.lg;
    if (props.md) return props.theme.fontSize.md;
    if (props.nm) return props.theme.fontSize.nm;
    if (props.sm) return props.theme.fontSize.sm;
    return 'inherit';
  },
  spacing: {
    xl: '8em',
    lg: '4em',
    md: '2em',
    nm: '1em',
    sm: '0.5em',
    none: '0em'
  },
  mediaBreakpoints: {
    mobile: (...args) => css`@media (max-width: 32em) {${css(...args)}}`,
    tablet: (...args) => css`@media (max-width: 72em) {${css(...args)}}`,
    desktop: (...args) => css`@media (min-width: 72em) {${css(...args)}}`
  },
  shadow: '#cfcfcf 0.1px 0.1px 3px 0px',
  nav: {
    height: '70px'
  }
};

injectGlobal`
  .agilysys-icon-new {
    color: ${theme.colors.new}
  }
  .agilysys-icon-healthy {
    color: ${theme.colors.healthy}
  }
  .agilysys-icon-vegetarian {
    color: ${theme.colors.vegetarian}
  }
  .agilysys-icon-vegan {
    color: ${theme.colors.vegan}
  }
  .agilysys-icon-spicy {
    color: ${theme.colors.spicy}
  }
  .agilysys-icon-gluten-free {
    color: ${theme.colors.glutenFree}
  }
  @media print {
    @-moz-document url-prefix() {
      .appContainer, .appContainer>div, .appContainer .parent, .appContainer .BottomContainer {
        display: block !important;
      }
    }
  }
`;

export default theme;
