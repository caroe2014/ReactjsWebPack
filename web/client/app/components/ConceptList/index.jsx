// Copyright Â© 2018-2019 Agilysys NV, LLC.  All Rights Reserved.  Confidential Information of Agilysys NV, LLC.
/* eslint-disable max-len */
import React, { Component } from 'react';
import styled, { ThemeProvider } from 'styled-components';
import theme from 'web/client/theme';
import PropTypes from 'prop-types';
import { Flex, Heading, Text } from 'rebass';
import Tile from 'web/client/app/components/Tile';
import { Trans } from 'react-i18next';
import i18n from 'web/client/i18n';

const Container = styled(Flex)`
  flex-direction: column;
  margin: 120px auto 0px auto;
  height: fit-content;
  max-width: 75em;
  width: 100%;
  padding: 0px;
  &:after{
  content: '';
  margin: auto;
  width: 100%;
  max-width: 570px;
  }
  ${props => props.theme.mediaBreakpoints.mobile`
    justify-content: center;
    padding-left: 10px;
    padding-right: 10px;
  `}
  ${props => props.theme.mediaBreakpoints.tablet`
    justify-content: center;
  `}
  ${props => props.theme.mediaBreakpoints.desktop`
    justify-content: space-between;
    padding-left: 20px;
    padding-right: 20px;
  `}
`;

const DigitalMenuError = styled(Container)`
  padding-top: 100px;
  ${props => props.theme.mediaBreakpoints.mobile`
    padding-top: 60px;
  `}
`;

const ListContainer = styled(Flex)`
  flex-direction: column;
  flex-flow: row wrap;
  height: fit-content;
  max-width: 75em;
  width: 100%;
  padding: 0px;
  &:after{
  content: '';
  margin: auto;
  width: 100%;
  max-width: 570px;
  }
  ${props => props.theme.mediaBreakpoints.mobile`
    justify-content: center;
  `}
  ${props => props.theme.mediaBreakpoints.tablet`
    justify-content: center;
  `}
  ${props => props.theme.mediaBreakpoints.desktop`
    justify-content: space-between;
  `}
  &:focus {
    outline: none;
    box-shadow: none;
  }
`;

const Toptext = styled(Heading)`
  width: 100%;
  color: ${props => props.theme.colors.primaryTextColor};
  font-size: 18px;
  margin-top: 20px;
  margin-bottom: 20px;
  font-weight: 700;
  ${props => props.theme.mediaBreakpoints.tablet`margin: 20px auto; max-width: 570px;`}
  &:focus {
    outline: none;
    box-shadow: none;
  }
`;

const ErrorHeading = styled(Text)`
  text-align: center;
  font-weight: bold;
  font-size: 24px;
  color: ${props => props.theme.colors.primaryTextColor};
`;

const ErrorText = styled(Text)`
  text-align: center;
  font-weight: 400;
  font-size: 18px;
  color: ${props => props.theme.colors.primaryTextColor};
`;

const OpenTextContainer = styled(Flex)`
  flex-direction: column;
  position: relative;
  font-size: 14px;
  margin-top: 8px;
  z-index: 10;
  font-weight: 500;
`;

const TitleContainer = styled(Flex)`
  flex-direction: column;
`;

const TimingParent = styled.span`
  margin: auto;
  text-align: center;
  padding-top: 8px;
`;

const TimingMessage = styled.span`
  color: ${props => props.theme.colors.secondaryTextColor};
  font-size: 14px;
  &:focus {
    outline: none;
    box-shadow: none;
  }
`;

const TimingText = styled.span`
  color: ${props => props.theme.colors.secondaryTextColor};
  font-size: 14px;
  &:focus {
    outline: none;
    box-shadow: none;
  }
`;

const openIcon = {
  color: theme.colors.open,
  fontSize: '16px'
};

const closeIcon = {
  color: theme.colors.close,
  fontSize: '16px'
};

class ConceptList extends Component {
  constructor (props) {
    super(props);
    this.handleOnClick = this.handleOnClick.bind(this);
  }
  componentWillMount () {
    this.props.digitalMenuId && this.props.keyProps.length === 1 && this.handleOnClick(this.props.keyProps[0].id);
  }
  handleOnClick (conceptId) {
    if (this.props.selectConcept) {
      this.props.selectConcept(conceptId);
    }
  }
  getTimingMessage (availableAt, isOpen) {
    if (availableAt) {
      let hour = parseInt(availableAt.time / 60);
      let minutes = availableAt.time % 60;
      if (isOpen) {
        return hour === 0 ? i18n.t('CONCEPT_CLOSES_IN', { minutes: minutes }) : i18n.t('CONCEPT_AVAILABLE_NOW');
      } else {
        const conceptTag = 'CONCEPT_AVAILABLE';
        const hourTag = hour > 0 ? '_HRS' : '';
        const minuteTag = minutes > 0 ? '_MIN' : '';
        return i18n.t(`${conceptTag}${hourTag}${minuteTag}`, { hours: hour, minutes: minutes });
      }
    } else {
      return i18n.t('CLOSED_NOW');
    }
  }
  render () {
    const { keyProps, isScheduleOrderEnabled, showOperationTimes, digitalMenuId, siteList, digitalMenuError } = this.props;
    const closedScreenInfo = digitalMenuError && siteList && siteList.length === 1 && siteList[0].closedScreen;
    return (
      <ThemeProvider theme={theme}>
        <React.Fragment>
          {keyProps.length === 0 && digitalMenuError &&
          <DigitalMenuError>
            <ErrorHeading
              tabIndex={0}
              aria-label={(closedScreenInfo && closedScreenInfo.headerText) || i18n.t('DIGITAL_MENU_NO_MENU_TITLE')}>
              {(closedScreenInfo && closedScreenInfo.headerText) || i18n.t('DIGITAL_MENU_NO_MENU_TITLE')}
            </ErrorHeading>
            <ErrorText
              tabIndex={0}
              aria-label={(closedScreenInfo && closedScreenInfo.instructionText) || i18n.t('DIGITAL_MENU_NO_MENU_INSTN')}>
              {(closedScreenInfo && closedScreenInfo.instructionText) || i18n.t('DIGITAL_MENU_NO_MENU_INSTN')}
            </ErrorText>

          </DigitalMenuError>
          }
          {keyProps.length > 0 &&
          <Container className='concept-list' p={[1, 2, 3, 4]}>
            <Toptext className='select-concept-header' role='heading' tabIndex={0} aria-level='2'>
              {!digitalMenuId && <Trans i18nKey='CONCEPT_LIST_PAGE_LABEL' />}</Toptext>
            <ListContainer className='concept-list-container'
              tabIndex={0}
              aria-label={`${(keyProps && keyProps.length > 1)
                ? `${i18n.t('CONCEPT_SELECT')} ${keyProps.length} ${i18n.t('CONCEPTS')}` : ``}`}>
              {keyProps && keyProps.map((item, index) => {
                let title = <TitleContainer>
                  <Text>{item.name}</Text>
                  <OpenTextContainer className='opens-text'>
                    {!digitalMenuId && !isScheduleOrderEnabled && showOperationTimes &&
                    <TimingMessage className='custom-title'>
                      {this.getTimingMessage(item.availableAt, item.availableNow)}
                    </TimingMessage>
                    }
                    {!digitalMenuId && showOperationTimes && item.availableAt && <TimingParent>
                      <i className='fa fa-clock-o'
                        style={item.availableNow ? openIcon : closeIcon}
                        alt=''
                      />
                      <TimingText
                        isopen={item.availableNow}
                        className='custom-time'
                        tabIndex={0}
                        aria-label={item.availableNow ? `${i18n.t('AVAILABLE_FROM')} ${item.availableAt.open} ${i18n.t('TO_LABEL')} 
                      ${item.availableAt.close}` : `${i18n.t('NOT_AVAILABLE_FROM')} ${item.availableAt.open} ${i18n.t('TO_LABEL')}
                      ${item.availableAt.close}`}
                      >
                        {` ${item.availableAt.open} - ${item.availableAt.close}`}
                      </TimingText>
                    </TimingParent>}
                  </OpenTextContainer>
                </TitleContainer>;

                return <React.Fragment key={`concept-${item.id}`}>
                  <Tile
                    classContext={`tile concept-${index + 1} concept_${item.id}`}
                    className='tile'
                    imageresponsive='true'
                    title={title}
                    titleStyle={{ width: '100%', textAlign: 'center' }}
                    image={item.image}
                    id={item.id}
                    handleClick={this.handleOnClick}
                    isEnabled={item.availableNow}
                    closedoverlay={!digitalMenuId}
                    ariaLabel={`concept${index + 1} ${item.name}`}
                    tabIndex={0}
                  />
                </React.Fragment>;
              })}
            </ListContainer>
          </Container>
          }
        </React.Fragment>
      </ThemeProvider>
    );
  }
}

ConceptList.propTypes = {
  selectConcept: PropTypes.func,
  keyProps: PropTypes.arrayOf(
    PropTypes.shape({
      image: PropTypes.string,
      id: PropTypes.any.isRequired,
      name: PropTypes.string.isRequired
    }).isRequired
  )
};
export default ConceptList;
