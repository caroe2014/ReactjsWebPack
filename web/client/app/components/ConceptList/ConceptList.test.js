// Copyright Â© 2018-2019 Agilysys NV, LLC.  All Rights Reserved.  Confidential Information of Agilysys NV, LLC.

import React from 'react';
import { shallow, mount, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import ConceptList from '.';
import { ThemeProvider } from 'styled-components';
import theme from 'web/client/theme.js';

configure({ adapter: new Adapter() });

jest.mock('web/client/i18n', () => ({
  t: (key) => {
    return key;
  }
}));
jest.mock('react-i18next', () => ({
  // this mock makes sure any components using the translate HoC receive the t function as a prop
  withTranslation: () => Component => {
    Component.defaultProps = { ...Component.defaultProps, t: (key) => key };
    return Component;
  },
}));

jest.mock('react-i18next', () => ({
  Trans: 'div'
}));

describe('ConceptList', () => {

  let wrapper;
  let mockProps = [
    {
      image: 'https://www.freeiconspng.com/uploads/environment-icon-png-16.png',
      id: '101',
      name: 'Pickle',
      availableNow: true,
      availableAt: {
        time: 300
      }
    },
    {
      image: 'https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG',
      id: '102',
      name: 'Ice Berg',
      availableNow: false,
      availableAt: {
        time: 30
      }
    },
    {
      image: 'https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG',
      id: '108',
      name: 'Ice Berg',
      availableNow: true,
      availableAt: {
        time: 30
      }
    },
    {
      image: 'https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG',
      id: '109',
      name: 'Ice Berg',
      availableNow: false,
      availableAt: {
        time: 330
      }
    },
    {
      image: 'https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG',
      id: '110',
      name: 'Ice Berg',
      availableNow: false,
      availableAt: {
        time: 300
      }
    },
    {
      image: 'https://www.freeiconspng.com/uploads/chicken-caesar-salad-png-23.png',
      id: '103',
      name: 'Go Green',
      availableNow: true
    },
    {
      image: 'https://bakingmischief.com/wp-content/uploads/2016/05/chicken-shawarma-with-yogurt-sauce-image-square1.jpg', // eslint-disable-line max-len
      id: '104',
      name: 'Made Nice',
      availableNow: true
    }
  ];

  beforeEach(() => {
    wrapper = shallow(<ConceptList keyProps={mockProps} />
    );
  });

  it('does a snapshot test', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should render without throwing an error', () => {
    expect(wrapper.exists()).toBe(true);
  });

  it('should render without throwing an error ob mount', () => {
    const mountWrapper = mount(
      <ThemeProvider theme={theme}>
        <ConceptList keyProps={mockProps} showOperationTimes/>
      </ThemeProvider>
    );
    expect(mountWrapper.exists()).toBe(true);
  });

  describe('when Item props are received', () => {
    it('should render the same number of tiles', () => {
      expect(wrapper.find('.tile').length).toBe(mockProps.length);
    });
  });

  describe('when Item props are not received', () => {
    beforeEach(() => {
      mockProps = [];
    });
    it('should not render ConceptList', () => {
      const itemWrapper = shallow(<ConceptList keyProps={mockProps} />);
      expect(itemWrapper.find('.tile').length).toBe(0);
    });
  });

});
