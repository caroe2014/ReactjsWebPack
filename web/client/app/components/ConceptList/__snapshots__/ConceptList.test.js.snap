// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConceptList does a snapshot test 1`] = `
<ThemeProvider
  theme={
    Object {
      "colors": Object {
        "addOnTab": "#347CFF",
        "addToCart": "#347CFF",
        "addonBg": "#EEEEEE",
        "backgroundGrey": "#E6E6E6",
        "black": "#000000",
        "close": "red",
        "contextBackground": "#f0f0f3",
        "darkGrey": "#464646",
        "disableBackground": "#E4E4E4",
        "disableButton": "#D1D1D1",
        "error": "#cc0000",
        "footer": "#48a942",
        "glutenFree": "#8d7356",
        "healthy": "#359919",
        "hint": "#a6a6a6",
        "inputBottomBorder": "lightgray",
        "itemBorder": "lightgrey",
        "itemTile": "#3C474C0",
        "light": "white",
        "loader": "#ababab",
        "loginBackground": "#303030",
        "loginBoxColor": "#434149",
        "loginField": "#AAA7AC",
        "loginFormBackground": "#484848",
        "navigationMenu": "#48a942",
        "navigationMenuHover": "cornflowerblue",
        "new": "#ff6000",
        "open": "#48a942",
        "paymentSuccessTitle": "#505050",
        "placeHolder": "#909090",
        "primary": "#347CFF",
        "primaryButton": "#347CFF",
        "primaryText": "#383838",
        "secondaryText": "#a6a6a6",
        "spicy": "#ed1c24",
        "text": "black",
        "textGrey": "#505050",
        "tile": "#505050",
        "validationError": "#cc0000",
        "vegan": "#0d4a22",
        "vegetarian": "#0d4a22",
      },
      "fontFamily": "Roboto",
      "fontSize": Object {
        "lg": "1.8rem",
        "md": "1.3rem",
        "nm": "1rem",
        "sm": "0.75rem",
        "xl": "2.4rem",
      },
      "fontSizeFromProps": [Function],
      "mediaBreakpoints": Object {
        "desktop": [Function],
        "mobile": [Function],
        "tablet": [Function],
      },
      "nav": Object {
        "height": "70px",
      },
      "receiptFontFamily": "Roboto",
      "shadow": "#cfcfcf 0.1px 0.1px 3px 0px",
      "spacing": Object {
        "lg": "4em",
        "md": "2em",
        "nm": "1em",
        "none": "0em",
        "sm": "0.5em",
        "xl": "8em",
      },
    }
  }
>
  <Styled(Flex)
    className="concept-list"
    p={
      Array [
        1,
        2,
        3,
        4,
      ]
    }
  >
    <Styled(Heading)
      aria-level="2"
      className="select-concept-header"
      role="heading"
      tabIndex={0}
    >
      <div
        i18nKey="CONCEPT_LIST_PAGE_LABEL"
      />
    </Styled(Heading)>
    <Styled(Flex)
      aria-label="CONCEPT_SELECT 7 CONCEPTS"
      className="concept-list-container"
      tabIndex={0}
    >
      <Tile
        ariaLabel="concept1 Pickle"
        classContext="tile concept-1 concept_101"
        className="tile"
        closedoverlay={true}
        handleClick={[Function]}
        hasBorder={true}
        id="101"
        image="https://www.freeiconspng.com/uploads/environment-icon-png-16.png"
        imageresponsive="true"
        isEnabled={true}
        tabIndex={0}
        title={
          <Styled(Flex)>
            <Text
              m={0}
            >
              Pickle
            </Text>
            <Styled(Flex)
              className="opens-text"
            />
          </Styled(Flex)>
        }
        titleStyle={
          Object {
            "textAlign": "center",
            "width": "100%",
          }
        }
      />
      <Tile
        ariaLabel="concept2 Ice Berg"
        classContext="tile concept-2 concept_102"
        className="tile"
        closedoverlay={true}
        handleClick={[Function]}
        hasBorder={true}
        id="102"
        image="https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG"
        imageresponsive="true"
        isEnabled={false}
        tabIndex={0}
        title={
          <Styled(Flex)>
            <Text
              m={0}
            >
              Ice Berg
            </Text>
            <Styled(Flex)
              className="opens-text"
            />
          </Styled(Flex)>
        }
        titleStyle={
          Object {
            "textAlign": "center",
            "width": "100%",
          }
        }
      />
      <Tile
        ariaLabel="concept3 Ice Berg"
        classContext="tile concept-3 concept_108"
        className="tile"
        closedoverlay={true}
        handleClick={[Function]}
        hasBorder={true}
        id="108"
        image="https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG"
        imageresponsive="true"
        isEnabled={true}
        tabIndex={0}
        title={
          <Styled(Flex)>
            <Text
              m={0}
            >
              Ice Berg
            </Text>
            <Styled(Flex)
              className="opens-text"
            />
          </Styled(Flex)>
        }
        titleStyle={
          Object {
            "textAlign": "center",
            "width": "100%",
          }
        }
      />
      <Tile
        ariaLabel="concept4 Ice Berg"
        classContext="tile concept-4 concept_109"
        className="tile"
        closedoverlay={true}
        handleClick={[Function]}
        hasBorder={true}
        id="109"
        image="https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG"
        imageresponsive="true"
        isEnabled={false}
        tabIndex={0}
        title={
          <Styled(Flex)>
            <Text
              m={0}
            >
              Ice Berg
            </Text>
            <Styled(Flex)
              className="opens-text"
            />
          </Styled(Flex)>
        }
        titleStyle={
          Object {
            "textAlign": "center",
            "width": "100%",
          }
        }
      />
      <Tile
        ariaLabel="concept5 Ice Berg"
        classContext="tile concept-5 concept_110"
        className="tile"
        closedoverlay={true}
        handleClick={[Function]}
        hasBorder={true}
        id="110"
        image="https://www.milkmaid.in/Images/Recipe/Chocolate%20694x400_11.JPG"
        imageresponsive="true"
        isEnabled={false}
        tabIndex={0}
        title={
          <Styled(Flex)>
            <Text
              m={0}
            >
              Ice Berg
            </Text>
            <Styled(Flex)
              className="opens-text"
            />
          </Styled(Flex)>
        }
        titleStyle={
          Object {
            "textAlign": "center",
            "width": "100%",
          }
        }
      />
      <Tile
        ariaLabel="concept6 Go Green"
        classContext="tile concept-6 concept_103"
        className="tile"
        closedoverlay={true}
        handleClick={[Function]}
        hasBorder={true}
        id="103"
        image="https://www.freeiconspng.com/uploads/chicken-caesar-salad-png-23.png"
        imageresponsive="true"
        isEnabled={true}
        tabIndex={0}
        title={
          <Styled(Flex)>
            <Text
              m={0}
            >
              Go Green
            </Text>
            <Styled(Flex)
              className="opens-text"
            />
          </Styled(Flex)>
        }
        titleStyle={
          Object {
            "textAlign": "center",
            "width": "100%",
          }
        }
      />
      <Tile
        ariaLabel="concept7 Made Nice"
        classContext="tile concept-7 concept_104"
        className="tile"
        closedoverlay={true}
        handleClick={[Function]}
        hasBorder={true}
        id="104"
        image="https://bakingmischief.com/wp-content/uploads/2016/05/chicken-shawarma-with-yogurt-sauce-image-square1.jpg"
        imageresponsive="true"
        isEnabled={true}
        tabIndex={0}
        title={
          <Styled(Flex)>
            <Text
              m={0}
            >
              Made Nice
            </Text>
            <Styled(Flex)
              className="opens-text"
            />
          </Styled(Flex)>
        }
        titleStyle={
          Object {
            "textAlign": "center",
            "width": "100%",
          }
        }
      />
    </Styled(Flex)>
  </Styled(Flex)>
</ThemeProvider>
`;
